web-registry/src/hooks/useEcocreditQuery.ts

```diff
@@ -221,6 +221,7 @@ export default function useEcocreditQuery<T extends EcocreditQueryResponse>({
     project,
     batchesByProject,
     batchesByIssuer,
+    projectsByClass,
```

Comment:
this dependency `projectsByClass` was already registered and seems to have caused the PR build to fail ;P

Reply:
@mhagel I just pushed this fix

Reply:
also updated snapshots

Reply:
Thanks. I must have missed that in resolving merge conflicts